### DISCUSSION TITLE
Full screen overlay

### DISCUSSION THREAD

**Discussion by RollB22** [SauceLLC/sauce4zwift#74] [General]
This has probably been considered but love Sauce4zwift but hate having toolbar at bottom that does not auto hide even with reset in tak manager Win10 and running zwift in a window causes 4g dropouts and stuttering (no freesync on monitor) 
I wonder if 4g or 5g when available will be enough to run borderless zwifting via the 3rd party provisions. 
Not sure of my way forward as can't run sauce to any effect in busy rides or races without issues so have to go back to full screen - be great if I could run it not as an overlay but on phone or iPad or additional screen and the run zwift in full screen ?

**Comment by mayfield** [comment]
Hi @RollB22 Have you tried the Sauce built-in web server?  You can pull up just about all the windows with any web browser at http://127.0.0.1:1080 . So Sauce can run in the background (or even a different machine) and you can pull up the windows you like on the other device.  There is even a link to this in the main settings page.

You might also join the discord server because another user has made a Mod that lets you have a full dashboard of web based widgets specifically for 2nd screen type applications.

**Comment by RollB22** [comment]
Cheers
Thank you so much
Mark

**Comment by DaiC1** [comment]
Hi Guys looking for some help:

I run Zwift on a Mac mini M2 and installed the Saurce app, but when I run both, I can't get the Source overlay to appear in the Zwift window. I'm quite new to Mac OS, so I'm hoping someone can tell me what I'm doing wrong.

**Reply by mayfield** [reply]
Hi, make sure Zwift is in "windowed" mode in its' settings.  Otherwise the OS won't let other windows overlay it.

**Reply by DaiC1** [reply]
Thanks for the prompt response. I'm gutted it can't be used full screen.
